src/browser/tab/mod.rs

```diff
@@ -723,47 +763,50 @@ impl<'a> Tab {
         Ok(script_coverages)
     }
 
+    /// Enables fetch domain.
+    pub fn enable_fetch(
+        &self,
+        patterns: Option<&[fetch::methods::RequestPattern]>,
+        handle_auth_requests: Option<bool>,
+    ) -> Fallible<&Self> {
+        self.call_method(fetch::methods::Enable {
+            patterns,
+            handle_auth_requests,
+        })?;
+        Ok(self)
+    }
+
+    /// Disables fetch domain
+    pub fn disable_fetch(&self) -> Fallible<&Self> {
+        self.call_method(fetch::methods::Disable {})?;
```

Comment:
If `Disable` has no members you can leave off it's `{}` in it's definition and creation here.

Reply:
(I see this is how the rest of the code is written, so feel free to ignore)